juniper/src/executor_tests/introspection/mod.rs

```diff
@@ -22,20 +21,22 @@ enum Sample {
 
 struct Scalar(i32);
 
-#[graphql_scalar(name = "SampleScalar")]
-impl<S: ScalarValue> GraphQLScalar for Scalar {
-    fn resolve(&self) -> Value {
-        Value::scalar(self.0)
+#[graphql_scalar(with = sample_scalar, parse_token = i32)]
+type SampleScalar = Scalar;
```

Comment:
Let's allow only `parse_token(i32)` argument form. `parse_token = i32` is kinda confusing.